README.md

```diff
@@ -6,40 +6,40 @@ User application for Agoric Inter Protocol--Vaults, BLD Boost, Liquidations, etc
 
 1. Download and build the latest copy of `agoric-sdk`.
 
-```
-cd agoric-sdk
-yarn && yarn build
-```
+   ```
+   cd agoric-sdk
+   yarn && yarn build
+   ```
 
 2. (One-time) Define keys expected by the startup scripts
 
-```
-agd keys add gov1 --keyring-backend=test
-agd keys add gov2 --keyring-backend=test
-```
+   ```
+   agd keys add gov1 --keyring-backend=test
+   agd keys add gov2 --keyring-backend=test
+   ```
 
-If you have access to the mnemonics, use `--interactive` to get a prompt to import them. Without that it will create new keys. You can save the seed phrase somewhere, or even create a keplr account with it for testing. You can use `agd keys list --keyring-backend=test` to check which keys you've created.
+   If you have access to the mnemonics, use `--interactive` to get a prompt to import them. Without that it will create new keys. You can save the seed phrase somewhere, or even create a keplr account with it for testing. You can use `agd keys list --keyring-backend=test` to check which keys you've created.
 
 3. Start a local chain with psm, vaults, etc.
 
-```
-cd packages/inter-protocol && scripts/start-local-chain.sh
-```
+   ```
+   cd packages/inter-protocol && scripts/start-local-chain.sh
+   ```
 
-4. Start a local [wallet](https://github.com/Agoric/wallet-app) client server:
+4. Go the the [wallet](https://github.com/Agoric/wallet-app) repository and start a local client server:
 
-```
-cd wallet-app
-yarn start
-```
+   ```
+   cd wallet-app
+   yarn start
+   ```
 
-Go to settings and select localhost for your network
+5. Open the [wallet UI](http://localhost:3000/wallet/) and adjust Settings to use the localhost network.
 
-<img width="410" alt="Screen Shot 2023-01-23 at 11 57 16 AM" src="https://user-images.githubusercontent.com/8848650/214137346-b42942db-3b93-413a-991e-c77e2a30d6f1.png">
+   <img width="410" alt="Screen Shot 2023-01-23 at 11 57 16 AM" src="https://user-images.githubusercontent.com/8848650/214137346-b42942db-3b93-413a-991e-c77e2a30d6f1.png">
 
-If you created a Keplr account with the seed from YOUR_ACCOUNT_KEY, you should already have a smart wallet provisioned.
+   If you created a Keplr account with the seed from YOUR_ACCOUNT_KEY, you should already have a smart wallet provisioned.
 
-5. Then `yarn && yarn dev` in this directory to start the local HMR server. To connect to your local wallet UI, use http://127.0.0.1:5173/?wallet=local.
+6. Return to this repository and `yarn && yarn dev` to start the local HMR server. To connect to your local wallet UI, use http://127.0.0.1:5173/?wallet=local.
```

Comment:
What is "HMR"?

Reply:
"Hot Module Replacement". It's how the local dev server automatically updates the UI as you make changes without having to refresh